Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM python:3.9.6-slim-buster as parent
+
+RUN apt-get update && apt-get install -y curl && \
+    curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+
+ENV PATH="/root/.poetry/bin:${PATH}"
+
+WORKDIR /app
+
+COPY pyproject.toml /app/
+
+RUN poetry config virtualenvs.create false
+
+RUN  poetry install --no-dev --no-root
+
+COPY . /app
+
+EXPOSE 5000
+
+FROM parent as development 
+
+ENTRYPOINT poetry run flask run --host=0.0.0.0
```

Comment:
Minor, but worth noting that you've only installed the non-dev dependencies above - here that's fine (since you aren't tracking any dev dependencies currently) but worth noting that we might want to install any remaining dependencies as part of the development target